src/main/java/uk/gov/hmcts/ccd/auditlog/AuditOperationType.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.ccd.auditlog;
+
+public enum AuditOperationType {
+    CREATE_CASE("Create case"),
+    UPDATE_CASE("Update case"),
+    CASE_ACCESSED("Case Accessed"),
+    SEARCH_CASE("Search case"),
+    UPDATE_CASE_ACCESS("Update case access"),
```

Comment:
mmm did we say to call this "Update case access permissions"?

Reply:
done